.vscode/settings.json

```diff
@@ -3,5 +3,9 @@
   "files.exclude": {
     "**/*.element.css.ts": true,
     "**/*.global.css.ts": true
-  }
-}
\ No newline at end of file
+  },
+  "i18n-ally.localesPaths": [
+    "packages/core/src/internal/i18n",
+    "packages/angular/projects/clr-angular/src/utils/i18n"
+  ]
```

Comment:
Should this be defined in the project repo or per local machine?

Reply:
Tried to removed it but VSCode constantly generated it after I tried delete it. Strange behaviour but didn't dig into what's causing it.